### ISSUE TITLE
[help] Request to join to public gumbo project

### ISSUE THREAD

**Issue by txgk** [sparklemotion/nokogiri#2930]
**What problem are you trying to solve?**

Hello, nokogiri developers! I recently started to maintain a [public repository for gumbo-parser](https://codeberg.org/grisha/gumbo-parser) because I saw that a lot of projects do their own thing since official GitHub repository for gumbo-parser lost maintenance. I faced this issue myself, wanting to vendor in the gumbo code and start adding on top, but thought it would be cool to start gumbo anew and make it public like it was more than 7 years ago. So I did it and sent this message to all maintainers of gumbo-parser repository packages:

```
Hello, dear gumbo-parser package maintainer.

I write to you because old gumbo-parser repository has been archived and marked
unmaintained by original author (actually it was unmaintained since 2016).

Fortunately I created fork which continues the ideas of the original project,
monitors bug reports and is being maintained. It's hosted on Codeberg (grisha/gumbo-parser).   

In the new release of gumbo-parser 0.11.0 I added support for picture tag and
made one of the build script compatible with Python 3. Also all email addresses
for bug reporting were changed to my email address.

Main reasoning behind this move is my use of gumbo-parser in one of the software
projects, which strives to develop and unmaintained gumbo-parser is a block on   
the way to this.

I humbly ask you to update gumbo-parser to 0.11.0 changing source repo to page on
Codeberg (grisha/gumbo-parser). Thank you very much :)

Best regards, Grigory Kirillov
```
In the first two hours, the package was updated only in Gentoo, but I think that there will be more migrations to the maintained project soon.

**Please show your code!**

All I have is a vanilla gumbo-parser with some tags support added. I understand that nokogiri_gumbo has undergone a lot of changes since it was forked, but I would like to humbly ask for a little help nevertheless. Since this new project must be somewhat close to original gumbo, we can't do things like dropping custom allocator support or changing handling of some special tags, but some optimizations might be very handy.

Thank you for taking your time to read all this ;)

**Comment by flavorjones** [comment]
cc @stevecheckoway

**Comment by flavorjones** [comment]
Hi, thanks for reaching out. Our gumbo fork is in this repository, in https://github.com/sparklemotion/nokogiri/tree/main/gumbo-parser

We have made minor changes, primarily to keep the parser compatible with HTML5 and https://github.com/html5lib/html5lib-tests.git, feel free to look at the project history and cherry-pick any commits that seem relevant.

Long term, I would like very much to help support the gumbo parser, however I'm spread a bit thin at the moment with my open-source time and I'm not sure how much time I'll have to dedicate to it in the next few months.

I've pinged @stevecheckoway who has done the most work keeping nokogiri's gumbo fork up-to-date. I know he's been in communication with other gumbo contributors. I'm sure he'll share his mental models here when he is able to find time.
